projects/kit/components/input-phone-international/tokens/countries-masks.ts

```diff
@@ -70,7 +70,7 @@ export const TUI_COUNTRIES_MASKS: InjectionToken<Record<TuiCountryIsoCode, strin
             [TuiCountryIsoCode.FK]: `+500#####`,
             [TuiCountryIsoCode.FM]: `+691###-####`,
             [TuiCountryIsoCode.FR]: `+33 ## ## ## ## ##`,
-            [TuiCountryIsoCode.GA]: `+241#-##-##-##`,
+            [TuiCountryIsoCode.GA]: `+241##-##-##-##`,
```

Comment:
`Gabon` format is `+241 XX XX XX XX` (+241 and 8 digits): https://en.wikipedia.org/wiki/Telephone_numbers_in_Gabon#:~:text=Calling%20formats&text=To%20call%20Gabon%2C%20the%20following,XX%20%2D%20calling%20from%20outside%20Gabon